api/types/container/host_config.go

```diff
@@ -313,7 +313,7 @@ type HostConfig struct {
 	Runtime         string            `json:",omitempty"` // Runtime to use with this container
 
 	// Applicable to Windows
-	ConsoleSize Box       // Initial console size
+	ConsoleSize [2]uint   // Initial console size (height,width)
```

Comment:
height, width looks backwards. Can we confirm ordering? This wasn't in the original PR.

(Really, everything I have ever seen does WxH, but if this is backwards, let's make sure.)